File: frontend/update_backend_video.js
const { PrismaClient } = require('@prisma/client');
const prisma = new PrismaClient();

async function main() {
  const targetEmbed = "https://www.youtube.com/embed/zfVnUHSsrUw";
  const lessons = await prisma.lesson.findMany({
    where: {
      title: { contains: 'HTTP & REST API' }
    }
  });

  console.log("Found lessons:", lessons.map(l => ({ id: l.id, title: l.title, slug: l.slug })));

  if (lessons.length > 0) {
    const updated = await prisma.lesson.update({
      where: { id: lessons[0].id },
      data: { video_url: targetEmbed }
    });
    console.log("Successfully updated:", updated.title, updated.video_url);
  }
}

main()
  .catch(e => console.error(e))
  .finally(() => prisma.$disconnect());

File: backend/update_backend_video.py
import asyncio
from prisma import Prisma

async def main() -> None:
    prisma = Prisma()
    await prisma.connect()

    try:
        lesson1 = await prisma.lesson.find_first(where={"slug": "backend-developer-lesson-1-1"})
        if lesson1:
            updated_lesson = await prisma.lesson.update(
                where={"id": lesson1.id},
                data={"video_url": "https://www.youtube.com/embed/zfVnUHSsrUw"}
            )
            print(f"Updated {updated_lesson.title} video_url to {updated_lesson.video_url}")

        lesson3 = await prisma.lesson.find_first(where={"slug": "backend-developer-lesson-1-3"})
        if lesson3:
            await prisma.lesson.update(
                where={"id": lesson3.id},
                data={"video_url": None}
            )
            print("Reverted lesson 3")

    finally:
        await prisma.disconnect()

if __name__ == "__main__":
    asyncio.run(main())
